keyvi/check-style.sh

```diff
@@ -8,7 +8,7 @@ cpplint_files=()
 for file in ${infiles}; do
   fqfile=`git rev-parse --show-toplevel`/${file}
   echo "Checking: ${file}"
-  if ! cmp -s ${fqfile} <(clang-format ${fqfile}); then
+  if ! cmp -s ${fqfile} <(clang-format -style=file ${fqfile}); then
```

Comment:
can you explain, is that something that does not work on your setup?

Reply:
At some point `clang-format` was not running, and it seemed to me that this had fixed it.
But just tested, it's fine even without it. Thx!